src/paket.dependencies

```diff
@@ -45,3 +45,12 @@ group VS2017
 	nuget Microsoft.TeamFoundationServer.ExtendedClient ~> 15.0
 	nuget Microsoft.VisualStudio.Setup.Configuration.Interop
 	nuget Microsoft.VisualStudio.Settings.15.0 ~> 15.0
+
+
+group VS2019
+	framework: net472
+	source https://api.nuget.org/v3/index.json
+	nuget Microsoft.TeamFoundationServer.Client ~> 16.0
+	nuget Microsoft.TeamFoundationServer.ExtendedClient ~> 16.0
+	nuget Microsoft.VisualStudio.Setup.Configuration.Interop
+	nuget Microsoft.VisualStudio.Settings.15.0 ~> 16.0
```

Comment:
😒 a library named "15.0" that's version "16.0". :shrug:

Reply:
Quite confusing, yes. 😄